travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+dist: trusty
+jdk:
+  - oraclejdk11
+
+addons:
+  sonarcloud:
+    organization: "aalto-letech-intellij-plugin"
+    token: $SONARCLOUD_TOKEN
+script:
+  - sonar-scanner
+branches:
+  - /.*/
```

Comment:
I guess this means build all braches. In <https://docs.travis-ci.com/user/customizing-the-build/#Skipping-the-Installation-Step> they use a bit longer form for that.
```
branches:
  only:
  - /.*/
```
I assume this is a proper shorthand as Travis build config is correctly constructed.

Reply:
travis build is triggered with a "workaround" (trigger build) feature of [travis](https://blog.travis-ci.com/2017-08-24-trigger-custom-build), so for now there has been no automated pick-ups from gh, which makes me puzzled

Reply:
FFS, it's **.travis.yml** (starting with a dot) but not **travis.yml**

Reply:
https://travis-ci.org/Aalto-LeTech/intellij-plugin/builds/634191182 like a charm